cura/Settings/MachineManager.py

```diff
@@ -337,16 +342,30 @@ def addMachine(self, name, definition_id):
     def _createUniqueName(self, container_type, current_name, new_name, fallback_name):
         return UM.Settings.ContainerRegistry.getInstance().createUniqueName(container_type, current_name, new_name, fallback_name)
 
-    ##  Convenience function to check if a stack has errors.
-    def _checkStackForErrors(self, stack):
+    ##  Convenience function to check if a stack is valid.
+    def _checkStackValid(self, stack):
```

Comment:
I think it is a good idea to move this to ContainerStack as a hasErrors() or similar (see also my other comment about the "valid" naming) rather than keeping it here. MachineManager is already way too overcrowded and this just adds more.